src/client/pages/Offer/Introduction/Sidebar/utils.ts

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
```

Comment:
This is just a copy of the same file placed inside Offer. Nothing was changed.

Reply:
Same, can we point to the old file instead to avoid fixing bugs or making changes in two places?